app/components/find/course_preview_missing_information_component.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Find
+  class CoursePreviewMissingInformationComponent < ViewComponent::Base
```

Comment:
Since this is shared, i'm tempted we make a new directory for the course preview or just use the existing `find/courses` then your module would be `Find::Courses::MissingInformationComponent` or just `CoursePreview::MissingInformationComponent`

Reply:
I think that's a sensible idea! 

https://github.com/DFE-Digital/publish-teacher-training/pull/3441/commits/77e8ef5a1083a5661e10f40e46f34b73d0afcf5c